vernac/mltop.ml

```diff
@@ -334,23 +333,38 @@ let _ =
 
 (* Liboject entries of declared ML Modules *)
 
+(* Digest of module used to compile the file *)
+type ml_module_digest =
+  | None
```

Comment:
This is going to be a pain if we use options in this file, either disable warning 40 or rename this constructor